src/components/Chat/Chat.tsx

```diff
@@ -63,89 +77,124 @@ export class Chat extends React.Component<ChatProps> {
     }
   };
 
-  formatMessage = (cmd: string, msg: string): React.ReactNode | string => {
-    if (cmd === 'host') {
-      return (
-        <React.Fragment>
-          {`changed the video to `}
-          <span style={{ textTransform: 'initial' }}>
-            {this.props.getMediaDisplayName(msg)}
-          </span>
-        </React.Fragment>
-      );
-    } else if (cmd === 'seek') {
-      return `jumped to ${formatTimestamp(msg)}`;
-    } else if (cmd === 'play') {
-      return `started the video at ${formatTimestamp(msg)}`;
-    } else if (cmd === 'pause') {
-      return `paused the video at ${formatTimestamp(msg)}`;
+  formatMessage = (cmd: string, msg: any): React.ReactNode | string => {
+    switch (cmd) {
```

Comment:
Can we keep if/else here? Sometimes switch/case leads to errors if breaks/returns are left out, so I kind of like the explicit braces.